### PR TITLE
Feature/parameters

### PR DESCRIPTION
**Pull request by rishabh-ti**

### CODE REVIEW

File: src/utils/code.ts [hunk 1]
```diff
@@ -30,6 +30,22 @@ type BuildBlockParams = {
   forceBuildBlock?: boolean
 }
 
+const buildParams = (paramsName: any) => {
```
Comment (vinayak-sachdeva) [inline src/utils/code.ts:33]: format it, use new lines and indentation.
or create an issue to do it later

Comment (rishabh-ti) [reply src/utils/code.ts:33]: > format it, use new lines and indentation. or create an issue to do it later

Added an issue for the same - [link](https://github.com/trilogy-group/tu2k22-ui-made-easy/issues/49)

File: src/hooks/useParamsForm.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { ChangeEvent, useCallback } from 'react'
+import { useSelector } from 'react-redux'
+import useDispatch from './useDispatch'
+import { getSelectedComponentId } from '~core/selectors/components'
+
+export const useParamsForm = () => {
+  const dispatch = useDispatch()
+  const componentId = useSelector(getSelectedComponentId)
+
+  const setValueFromEvent = ({
```
Comment (vinayak-sachdeva) [inline src/hooks/useParamsForm.ts]: where is it needed?

**Review by vinayak-sachdeva — APPROVED** [review]
Looks good. Please address the comments and merge it

File: src/hooks/useParamsForm.ts (hunk 2 above)
Comment (rishabh-ti) [reply src/hooks/useParamsForm.ts]: > where is it needed?

Removed! - [link](https://github.com/trilogy-group/tu2k22-openchakra/pull/2/commits/181e1dfe359f8165b2f806616491d18299a84574)

**Comment by rishabh-ti** [comment]
> Looks good. Please address the comments and merge it

`Squash and merge`?

**Comment by vinayak-sachdeva** [comment]
> > Looks good. Please address the comments and merge it
> 
> `Squash and merge`?

Normal merge, with merge commit

**Pull request merged**